opennms-webapp/src/main/webapp/element/node.jsp

```diff
@@ -442,305 +442,130 @@ function confirmAssetEdit() {
   
   <!-- Asset box, if info available --> 
   <c:if test="${! empty model.asset && (! empty model.asset.description || ! empty model.asset.comments)}">
-    <div class="panel panel-default">
-    <div class="panel-heading">
-      <h3 class="panel-title">Asset Information</h3>
-    </div>
-    <table class="table table-condensed">
-      <tr>
-        <th>Description</th>
-        <td>${model.asset.description}</td>
-      </tr>
-      
-      <tr>
-        <th>Comments</th>
-        <td>${model.asset.comments}</td>
-      </tr>
-    </table>
-    </div>
+  <div class="panel panel-default">
+      <div class="panel-heading"><h3 class="panel-title">Asset Information</h3></div>
+      <table class="table table-condensed">
+      <tr><th>Description</th><td>${model.asset.description}</td></tr>
+      <tr><th>Comments</th><td>${model.asset.comments}</td></tr>
+      </table>
+  </div>
   </c:if>
+  <!--End Asset box, if info available --> 
 
   <!-- SNMP box, if info available -->
   <c:if test="${! empty model.node.sysObjectId}">
-    <div class="panel panel-default">
-    <div class="panel-heading">
-      <h3 class="panel-title">SNMP Attributes</h3>
-    </div>
-    
+  <div class="panel panel-default">
```

Comment:
Can we avoid reformatting the tables which we did not alter as part of these changes?

Reply:
Jesse...this is very well verified and I use a more compact layout for the code
if you carefully look there only a compact representations of objects.....
but if you prefer I can redo the changes done

Reply:
I'm going to make the requested changes....now